docs/3.tutorials/examples/donation.md

```diff
@@ -211,7 +211,7 @@ To interact with the contract through the console, you can use the following com
 ```bash
 # Get donations 
 # Optional arguments for pagination
-near view donation.near-examples.testnet get_donations --args='{"from_index": "0","limit": "10"}'
```

Comment:
```bash
...
Error: Error: Querying failed: wasm execution failed with error: HostError(GuestPanic { panic_msg: "panicked at src/d
onation.rs:19:1:\nFailed to deserialize input from JSON.: Error(\"invalid type: string \\\"0\\\", expected u64\", lin
e: 1, column: 17)" }).
```

Reply:
this is due to `U128` -> `u64` change
https://github.com/near-examples/donation-examples/pull/5/files#diff-026e21e2f8a7826839b998ee3fa9e0db698d1c8e70b0cf233db948d98c23944fL65

Reply:
there's essentially 2 versions of contract, maybe unchanged doc is valid for https://github.com/near-examples/donation-examples/tree/main/contract-ts (maybe not)